### PR TITLE
Updates DSL version and Adds support for single experiment test (load) 

### LINKED ISSUE #402: Adjust Goal Types and Update Life Cycle Handling
`test-manager` needs to be updated to ensure it can handle single execution for `load`:
- add selection strategy for one experiment - `SingleExperimentSelectionStrategy`
- add validation strategy for one experiment - `SingleExperimentValidationStrategy`

### PR DESCRIPTION
**Pull request by jeppe-style**
As per title, and fixes #402. In addition:
- Adds dynamic creation of test bundles for testing. 
- Adds response message to InvalidTestBundleWebException.

### CODE REVIEW

**Comment by VincenzoFerme** [comment]
@jeppe-style a test failed https://app.wercker.com/BenchFlow/benchflow/runs/build-benchflow-test-manager/594fd344c0556700012b4db5?step=594fd3e742d5cc00014db14f

File: benchflow-test-manager/application/src/test/resources/data/definition/exhaustive_exploration/one-at-a-time/users-memory-environment/benchflow-test.yml [hunk 1]
```diff
@@ -55,36 +55,29 @@ sut:
       sut_ready_log_check: ready
 
     deployment:
-      camunda: serverA
-      db: serverB
+      camunda: grid
```
Comment (VincenzoFerme) [inline benchflow-test-manager/application/src/test/resources/data/definition/exhaustive_exploration/one-at-a-time/users-memory-environment/benchflow-test.yml]: Use anonymous server names. This applies to all the test files.

**Review by VincenzoFerme — CHANGES_REQUESTED** [review]

File: benchflow-test-manager/application/src/test/resources/data/definition/exhaustive_exploration/one-at-a-time/users-memory-environment/benchflow-test.yml (hunk 1 above)
Comment (jeppe-style) [reply benchflow-test-manager/application/src/test/resources/data/definition/exhaustive_exploration/one-at-a-time/users-memory-environment/benchflow-test.yml]: changed

**Comment by VincenzoFerme** [comment]
@jeppe-style see https://github.com/benchflow/benchflow/issues/402#issuecomment-312951673

**Pull request merged**